README.md

```diff
@@ -177,6 +177,20 @@ prevent shipping late on a Friday, but still want to allow marking merge request
 More than one embargo period can be specified. Any merge request assigned to her
 during an embargo period, will be merged in only once all embargoes are over.
 
+## Restricting the list of projects marge-bot considers
+
+By default marge-bot will work on all projects that she is a member of.
+Sometimes it is useful to restrict a specific instance of marge-bot to a subset
+of projects. You can specify a regexp that projects must match (anchored at the
+start of the string) with `--project-regexp`.
+
```

Comment:
@aschmolck I understand the "anchored at the start of the string" part as saying that the there  is an implicit `^` at the beginning of the regexp, so that `--project-regexp foo/bar` will not match project `cafoo/bar`. Is that the intended meaning? If so, where does this happen in the code?